File: week-2/cross.c
#include <stdio.h>

void secret_shape(int n)
{
   for(int i = 1; i <= (2*n-1); i++)
   {
     for(int j = 1; j <= (2*n-1); j++)
     {
       if(i == n || j == n) printf("*");
	   else if(j < n) printf(" ");
     }
     printf("\n");
   }
   printf("\n");
}

int main()
{
   int start = 1;
   int end = 5;
   for(int i = 1; i <= 5; i++)
   {
      secret_shape(i);
   }
   return 0;
}

File: week-2/cross.py
def secret_shape(n):
    for i in range(1,2*n):
        for j in range(1,2*n):
            if (i == n or j == n):
                print("*",end="")
            elif (j < n):
                print(" ",end="")
        print()

for i in range(1,6):
    secret_shape(i)
    print()
